src/common/SurgeStorage.cpp

```diff
@@ -628,11 +628,11 @@ void SurgeStorage::load_wt_wt(string filename, Wavetable* wt)
    memset(&wh, 0, sizeof(wt_header));
 
    size_t read = fread(&wh, sizeof(wt_header), 1, f);
-   // I'm not sure why this ever worked but it is checking the 4 bytes against vawt so...
-   // if (wh.tag != vt_read_int32BE('vawt'))
-   if (!(wh.tag[0] == 'v' && wh.tag[1] == 'a' && wh.tag[2] == 'w' && wh.tag[3] == 't'))
+   if (strcmp(wh.tag, "twav"))
```

Comment:
the original looked for "vawt" in tag 0-3 and yet you compare with "twav" - why? Certainly the case that the files start with "vawt"

```
paul:~/dev/music/surge$ od -c resources/data/wavetables/generated/pmfreq.wt  | head -1
0000000    v   a   w   t 200  \0  \0  \0      \0 004  \0  \0  \0   F 006
```

Reply:
Odd I'll test this today. Must have done something wrong or the change was not in the build. Anyway, after this PR I'll work on the items I've marked as FIXME in this commit.

Reply:
Just for info, ‘vawt ‘stands for ‘vember Audio wave table’ according to internets